test/integration/index.js

```diff
@@ -222,6 +222,33 @@ const customTemplatesExample = () => {
   })
 }
 
+const classesPropsExamples = () => {
+  describe.only('classes properties', () => {
```

Comment:
Just got a stray `.only()` here

```suggestion
  describe('classes properties', () => {
```

Reply:
Whoop, sorry for that 😬